server/channels/app/job.go

```diff
@@ -52,6 +52,14 @@ func (a *App) GetJobsByTypes(c request.CTX, jobTypes []string, offset int, limit
 	return jobs, nil
 }
 
+func (a *App) GetJobsByTypeAndStatus(c request.CTX, jobTypes []string, status string, page int, perPage int) ([]*model.Job, *model.AppError) {
```

Comment:
Similar comment as the other spot - context and an options struct would look cleaner

Reply:
I agree, I was about to push this change but Dylan has already tested this so I'm going to make those changes in a separate PR. I'll add it to this ticket https://mattermost.atlassian.net/browse/MM-58163